speech_evaluation/utterance_metrics/squim.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Jiatong Shi
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+import numpy as np
+import torch
+
+try:
+    from pesq import pesq
+    from pystoi import stoi
+    import torchaudio
+    import torchaudio.functional as F
+    from torchaudio.pipelines import SQUIM_OBJECTIVE, SQUIM_SUBJECTIVE
+except ImportError:
+    raise ImportError(
+        "Please install pesq, pystoi, torchaudio and retry: pip install stoi"
+    )
+
+
+def squim_metric(pred_x, gt_x, fs):
+
```

Comment:
For better reference, it would be great if you could add the corresponding reference paper/links as the docstring to this function.